### ISSUE TITLE
FEATURE: enable support for conditional inclusing of each tag

### ISSUE THREAD

**Issue by LongLiveCHIEF** [crazy-max/ghaction-docker-meta#28]
Opening this in https://github.com/HackerHappyHour/tagging-strategy/issues/42 as well

***

allow the use of `::true` or `::false` after each tag to specify whether or not that tag should be produced.

This will allow the use of expressions to determine whether or not a tag should be produced.

Here's a real example: you may want to produce the `{{ major }}` pattern under most conditions, but it doesn't
make sense to do so in the given tag is a pre-release.  You could do something like:

```yaml
-
  with:
    tags: |
      {{major}}::${{ github.event.action != 'prerelease' }}
      {{major}}.{{minor}}.{{patch}}
```

**Comment by LongLiveCHIEF** [comment]
closed in favor of #30

**Comment by LongLiveCHIEF** [comment]
I'm closing this since I implemented it with the latest release of HackerHappyHour/tagging-strategy, and it's getting to the point with this action IMO where it's too tightly coupled to the workflow event, with too much indirect control and output to really be capable of the flexibility I need.  Will open a separate issue with a proposal to redesign ghaction-docker-meta tagging abilities.
